fabtests/pytest/efa/efa_common.py

```diff
@@ -123,7 +123,7 @@ def get_efa_domain_names(server_id):
     process_timed_out = False
 
     # This command returns a list of EFA domain names and its related info
-    command = "ssh {} fi_info -p efa".format(server_id)
+    command = "ssh {} 'fi_info -p efa || /opt/amazon/efa/bin/fi_info -p efa'".format(server_id)
```

Comment:
I still think it is better to populate the environment variables from cmdline_args, which can include $PATH.